cmd/kubeadm/app/phases/certs/certlist.go

```diff
@@ -151,6 +152,7 @@ func (t CertificateTree) CreateTree(ic *kubeadmapi.InitConfiguration) error {
 				continue
 			}
 			// CA key exists; just use that to create new certificates.
+			fmt.Printf("[certs] Using the existing CA certificate %q and key %q\n", filepath.Join(ic.CertificatesDir, fmt.Sprintf("%s.crt", ca.BaseName)), filepath.Join(ic.CertificatesDir, fmt.Sprintf("%s.key", ca.BaseName)))
```

Comment:
```suggestion
			fmt.Printf("[certs] Using existing %s certificate authority\n", ca.BaseName)
```

as cert dir is already printed later in below.
https://github.com/kubernetes/kubernetes/blob/335fd41484a86bcd4d213924871eb1cabc6c6244/cmd/kubeadm/app/phases/certs/certs.go#L65-L69

Reply:
I added this line mostly want to let us know where is the CA certs exist, it's using the default location "/etc/kubernetes/pki/" or the dry-run directory "/etc/kubernetes/tmp/"? 

It was using "/etc/kubernetes/pki/" and this is why this is not enabled along with other sub-phases (https://github.com/kubernetes/kubernetes/pull/112945).

The full path will help us to debug if the following actions to create certs failed for some reason, but won't give any hint where the CA files located.

Reply:
got that. It seems to be different.

```
[download-certs] Downloading the certificates in Secret "kubeadm-certs" in the "kube-system" Namespace
[download-certs] Download the certificates to the folder: "/etc/kubernetes/tmp/kubeadm-join-dryrun2327321268"
[certs] Using certificateDir folder "/etc/kubernetes/pki"
I1013 14:31:43.122073    3706 certs.go:47] creating PKI assets
[certs] Using the existing CA certificate "/etc/kubernetes/tmp/kubeadm-join-dryrun2327321268/front-proxy-ca.crt" and key "/etc/kubernetes/tmp/kubeadm-join-dryrun2327321268/front-proxy-ca.key"
[certs] Generating "front-proxy-client" certificate and key
```

Reply:
Can you reach here https://github.com/kubernetes/kubernetes/blob/335fd41484a86bcd4d213924871eb1cabc6c6244/cmd/kubeadm/app/phases/certs/certs.go#L69
If any errors thrown from here: 
https://github.com/kubernetes/kubernetes/blob/335fd41484a86bcd4d213924871eb1cabc6c6244/cmd/kubeadm/app/phases/certs/certlist.go#L66

If yes, I agree with you.